ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/helpers/JavaHelper.java

```diff
@@ -157,9 +158,9 @@ public static String toXmlLine(
     }
 
     private static String indent(final int depth) {
```

Comment:
This method is being duplicated here and in the c autogen.  It should be moved to a common helper method.